src/engine/sys/sdl_input.cpp

```diff
@@ -1157,7 +1157,16 @@ static void IN_ProcessEvents( bool dropInput )
 					case SDL_WINDOWEVENT_RESTORED:
 					case SDL_WINDOWEVENT_MAXIMIZED:    Cvar_SetValue( "com_minimized", 0 ); break;
 					case SDL_WINDOWEVENT_FOCUS_LOST:   Cvar_SetValue( "com_unfocused", 1 ); break;
-					case SDL_WINDOWEVENT_FOCUS_GAINED: Cvar_SetValue( "com_unfocused", 0 ); break;
+					case SDL_WINDOWEVENT_FOCUS_GAINED: Cvar_SetValue( "com_unfocused", 0 );
+
+					                                   // HACK: if the window is focused, it can't be minimized.
```

Comment:
Is it just me or does the formatting here look completely wrong?

Reply:
Well, I guess it was meant to be aligned with `Cvar_SetValue`, but one tab would be enough instead (at least I would prefer a tab to this ascii-art).

Reply:
yes, it was for consistency with the other lines. Do you prefer if I put the other case on a newline for consistency?

Reply:
If you want to restyle other lines just for the sake of doing it, I would prefer doing another commit.

Reply:
I don't how to unmark the PR as "changes requested", but I changed that.